KamerConnect.DataAccess.Postgres/Repositories/MatchRepository.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Data.Common;
+using KamerConnect.Models;
+using KamerConnect.Repositories;
+using KamerConnect.Utils;
+using Npgsql;
+
+namespace KamerConnect.DataAccess.Postgres.Repositories;
+
+public class MatchRepository : IMatchRepository
+{
+    private readonly string connectionString;
+
+    public MatchRepository()
+    {
+        connectionString = EnvironmentUtils.GetConnectionString();
+    }
+    
+    public Match[]? GetMatchesByHouseId(Guid Houseid)
```

Comment:
Je mag aan de code te zien ook een personId meegeven kan je dit dan niet gewoon id noemen of iets?